package.json

```diff
@@ -21,23 +21,24 @@
   },
   "license": "MIT",
   "dependencies": {
-    "async": "~0.9.0",
-    "commondir": "0.0.1",
-    "debug": "^2.0.0",
-    "lodash": "^3.6.0",
-    "semver": "^4.1.0",
-    "toposort": "^0.2.10"
+    "async": "^0.9.2",
```

Comment:
While you are changing dependencies, could you please upgrade to the latest version? Either `^1.5.0` or `^2.0.0-rc.4`.